com.scoutshonour.dtipbijays.yaml

```diff
@@ -1,13 +1,13 @@
 app-id: com.scoutshonour.dtipbijays
 runtime: org.freedesktop.Platform
-runtime-version: '20.08'
+runtime-version: '22.08'
```

Comment:
Can you try 23.08?

Reply:
Would you mind taking over? I'm not too interested in stretching myself out more, again.

Reply:
Ok sure